from novaclient.exceptions import NotFound as NovaNotFound
from cinderclient.exceptions import NotFound as CinderNotFound